src/Modules/Routes/complaints-routes.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const userController = require('../Modules/Controllers/user-controller');
+const router = express.Router();
+
+const {
+    getAllComplaints,
+    createComplaint,
+    changeComplaint,
+    deleteComplaint
+} = require('../Modules/Controllers/user-controller');
```

Comment:
besides, the reception records cannot be located in the user-controller, they must be located in a separate controller